src/Maths/math.vector.ts

```diff
@@ -656,6 +656,17 @@ export class Vector2 {
         return center;
     }
 
+    /**

```

Comment:
can you fix conflicts and make sure that CEnter is using CenterToRef?

Reply:
Will be done by monday :)